.github/workflows/canary-sites-split.yml

```
@@ -50,8 +50,8 @@ jobs:
           repository_name: '${{ matrix.split_repository }}'
 
           # ↓ the user signed under the split commit
-          user_name: 'brian-ci'
-          user_email: 'brian.perry@pantheon.io'
+          user_name: 'decoupled-service-user'
```

```suggestion
          user_name: 'pantheon-decoupled-service-user'
```
the email and username aren't the same unfortunately, not sure if this matters.